src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -210,10 +210,8 @@ const BaseRuleDetails = ({
                   <StackItem>
                     <strong>{intl.formatMessage(messages.riskofchange)}</strong>
                   </StackItem>
-                  <StackItem
-                    className={`pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm`}
-                  >
-                    <span className="ins-c-rule-details__stackitem">
+                  <StackItem className="pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm">
```

Comment:
Should we change **alignCenterOverride** to comply with this PR?

Reply:
@mkholjuraev Yes, thanks!